tests/kafkatest/services/kafka/kafka.py

```diff
@@ -38,8 +38,7 @@
 class KafkaService(KafkaPathResolverMixin, JmxMixin, Service):
 
     PERSISTENT_ROOT = "/mnt"
-    STDOUT_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
-    STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "kafka.log")
+    STDOUT_STDERR_CAPTURE = os.path.join(PERSISTENT_ROOT, "server-start-stdout-stderr.log")
```

Comment:
What is the reasoning for the file name? Does this only capture stdout and stderr for the server start?